OJP/OJP_Trips.xsd

```diff
@@ -122,7 +122,7 @@
 					</xs:element>
 					<xs:element name="IncludeAlternativeOptions" type="xs:boolean" default="false" minOccurs="0">
 						<xs:annotation>
-							<xs:documentation>Whether alternative options should be presented as well. Mainly important for dominated journeys or in the case of ContinuousLegs the second-best route. Should be optimised for the user expectance (see. e.g https://theses.hal.science/tel-01848737). However, what the alternative options are may vary widely depending on the optimisation methods and filters.</xs:documentation>
+							<xs:documentation>Whether alternative options should be presented as well. Mainly important for dominated journeys or in the case of ContinuousLegs the second-best route. Should be optimized for user expectations (see. e.g https://theses.hal.science/tel-01848737). However, what the alternative options are may vary widely depending on the optimisation methods and filters.</xs:documentation>
```

Comment:
```suggestion
							<xs:documentation>Whether alternative options should be presented as well. Mainly important for dominated journeys or in the case of ContinuousLegs the second-best route. Should be optimised for user expectations (see. e.g https://theses.hal.science/tel-01848737). However, what the alternative options are may vary widely depending on the optimisation methods and filters.</xs:documentation>
```

Reply:
Britisher Englisher.

Reply:
https://en.wikipedia.org/wiki/Oxford_spelling

Reply:
I just realised (realized? ;-)) the usage is consistently with s instead of z in the rest of the document. Will push an update.